src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user;
+    private Storage storage;
+
+    @BeforeEach
+    public void setup() {
+        storage = new Storage();
+        user = new User();
+    }
+
+    @Test
+    void nullLogin_NotOk() {
+        user.setLogin(null);
+        assertThrows(OwnRuntimeException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void nullPassword_NotOk() {
+        user.setPassword(null);
+        assertThrows(OwnRuntimeException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void nullAge_NotOk() {
+        user.setAge(null);
+        assertThrows(OwnRuntimeException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void smallerThenEighteen_NotOk() {
+        user.setAge(17);
+        assertThrows(OwnRuntimeException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void registeredUser_NotOk() {
+        boolean actual = storage.people.contains(user.getLogin());
+        assertFalse(actual);
+    }
+
+    @Test
+    void passwordSmallerThenSix_NotOk() {
```

Comment:
```suggestion
    void passwordTooShort_NotOk() {
```
you may change the length limit at some point, and then have to deal with the hustle of renaming methods, Fix everywhere